spark/src/test/scala/io/delta/sharing/spark/DeltaSharingSourceSuite.scala

```diff
@@ -197,20 +184,23 @@ class DeltaSharingSourceSuite extends QueryTest
   /**
    * Test maxFilesPerTrigger and maxBytesPerTrigger
    */
-  integrationTest("maxFilesPerTrigger - success") {
-    val query = withStreamReaderAtVersion()
-      .option("maxFilesPerTrigger", "1")
-      .load().writeStream.format("console").start()
+  integrationTest("maxFilesPerTrigger - success with different values") {
+    Map(1 -> Seq(1, 1, 1, 1), 2 -> Seq(2, 2), 3 -> Seq(3, 1), 4 -> Seq(4), 5 -> Seq(4)).foreach{
```

Comment:
Consider adding a comment on what's in the map.

Reply:
added